Receipt/Sources/ListReceiptPresenter.swift

```diff
@@ -164,4 +250,25 @@ final class ListReceiptPresenter {
             }
         }
     }
+
```

Comment:
Update `let sortedGroupedSections = groupedSections
            .sorted(by: prepaidCardToken == nil ? { $0.key > $1.key } : { $0.key < $1.key })` to remove prepaidCardToken check so that receipts are always displayed in desc order

Reply:
And update `listPrepaidCardReceiptHandler` to pass the sorted (reversed) array to `groupReceiptsByMonth`